tests/integration/20.LB.Tests.ps1

```diff
@@ -107,6 +107,37 @@ Describe "Edge Load Balancer" {
 
     }
 
+    # Test to cover a bug where New-NsxLoadBalancerPool fails to create a new pool on the Edge,
+    # when Edge already has one existing pool and one existing VIP.
+    Context "Load Balancer Pool (Extended)" {
```

Comment:
Load Balancer Pool (Add Multiple Pool) ?

Reply:
Yeah, this bug happens only under certain circumstances, when:

1. You already have one pool
2. You already have one VIP
3. And you trying to add a second pool - that's when the bug manifests itself

I had a long post under #434 explaining why this happens.

This test is more for demonstration purposes and for confirming that the issue is fixed by the submitted commits. It's up to you if you want to keep this test moving forward, to make sure this bug never get reintroduced into the code again in future versions.

From pure testing purposes, it doesn't test anything more than the already existing "Load Balancer Pool" test.

Hope that makes sense.